packages/framework/src/parser.ts

```diff
@@ -348,9 +348,15 @@ const unwrapTheme = (theme?: string): EnsembleThemeModel | undefined => {
     },
   );
 
-  resolvedProps.forEach(([path, value]) =>
-    set(workingTheme.Styles!, path, value),
-  );
+  resolvedProps.forEach(([path, value]) => {
+    const stylePaths = path.split(".");
+    if (path.startsWith(".")) {
+      stylePaths.shift();
+      stylePaths[0] = `.${stylePaths[0]}`;
+    }
+
+    set(workingTheme.Styles!, [...stylePaths], value);
+  });
```

Comment:
I still don't think I understand what this is doing. Can you add an example here? Or even better, a unit test?

Reply:
this is very important for evaluate class styles with expressions
for example,

if theme file like,

```
{
    Styles: {
        .button-1: {
            backgroundColor: "${colors.primary['500']}"
        }
    }
}
```

and for evaluate this style correctly we need to convert them into
```
{
    Styles: {
        .button-1: {
            backgroundColor: "#ff0"
        }
    }
}
```

the logic here written is for set the value in object properties where key name starts with '.'

Reply:
How is it a complex expression? What happens if the expression of the style is `${someValue ?? otherValue}`